defaults/k8s_local_kind_aws_config.yaml

```diff
@@ -0,0 +1,2 @@
+instance_type_runner: c5.2xlarge
+root_disk_size_runer: 120
```

Comment:
Typo? Lost `n`?

Reply:
yes, and seem like I've copy pasted it around